### PR TITLE
refactor: replace walletconnect with bridge

### PR DESCRIPTION
**Pull request by igorosip0v**
Closes [WID-525](https://linear.app/worldcoin/issue/WID-525/world-id-bridge-is-implemented-in-simulator)

### What is done: 
- Removed walletconnect related code/packages
- Added endpoints/helpers for bridge support.
- Minor improvement

### Known issues
- ~`PUT {bridge_url}/response/{request_id}` returns `405 Method Not Allowed` ❗❗❗~
  - Resolved 
- ~`@worldcoin/idkit-core` still have issue with the extensions~
  - ~This is why test and build is broken~
  - Resolved

- ~Still using `idkit` 0.5.1 to generate external_nullifer~
  - Resolved

### Diagram

```mermaid
flowchart TD
    A[User opens Simulator] --> B[Press scan button];
    B --> C[Choose between scan and manual input];
    C --> D[Scanner recognizes url];
    D --> E[passing url to performVerification]
    E --> F["/api/pair-client"]
    F --> G["Getting data from 'GET {bridgeURL}/request/${requestUUID}'"]
    G --> H["return `{app_id,credential_type,action_description,action,signal}`"]
    H --> H1{Success?}
    H1 -->|no| If["reset process"]
    If --> If1["Show error"]
    H1 -->|yes| Is["Saving URL to store"]
    Is --> Is1["Generating fullProof"]
    Is1 --> J1["Receiving metadata from {dev_portal}/api/v1/precheck"]
    J1 --> J2["Saving metadata to storage"]
    J2 --> K["Showing modal with received metadata, credentials options and verify button"]
    K --> L["User selects credentials and press verify"]
    L --> M["/api/approve-request"]
    M --> N["validating + formatting payload + encrypt payload"]
    N --> O["PUT {bridgeURL}/request/${requestUUID}"]
    O --> P{Success?}
    P -->|yes| Ps["???"]
    P -->|no| Pf[???] 
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,36 +24,34 @@
     "@semaphore-protocol/group": "^3.9.0",
     "@semaphore-protocol/identity": "^3.9.0",
     "@semaphore-protocol/proof": "^3.9.0",
-    "@walletconnect/core": "^2.7.7",
-    "@walletconnect/utils": "^2.7.0",
-    "@walletconnect/web3wallet": "^1.7.5",
     "@worldcoin/idkit": "^0.4.9",
+    "@worldcoin/idkit-core": "1.0.0-alpha.0",
```
Comment (0xPenryn) [inline package.json:28]: once 1.0.0-alpha.1 is published we'll need to update this, but other than that this PR is good to go

File: src/lib/proof.ts
Comment (0xPenryn) [inline src/lib/proof.ts:1]: why doesn't this get an inclusion proof from the sequencer any longer? if we just generate a group and insert the single identity then proofs will never actually verify at all

**Review by 0xPenryn — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (igorosip0v) [reply package.json:28]: Sure!

**Review by 0xPenryn — APPROVED** [review]
should merge to a staging branch if possible rather than main, for the time being

**Pull request merged**